src/apps/material/helper.ts

```diff
@@ -117,3 +118,26 @@ export const getWorkDescriptionListData = ({
     }
   ];
 };
+
+export const getNoInterestAfter = (days: number) => {
```

Comment:
Great, I like it.
Although I think that we should use our configuration to store this map.
See: https://github.com/reload/dpl-react/blob/aben-modal-ved-tryk-pa-reserverknap-DDFSOEG-203/src/core/configuration/search.json#L10:L10